benchexec/container.py

```diff
@@ -531,29 +542,50 @@ def duplicate_mount_hierarchy(mount_base, temp_base, work_base, dir_modes):
             work_path = work_base + b"/" + str(overlay_count).encode()
             os.makedirs(temp_path, exist_ok=True)
             os.makedirs(work_path, exist_ok=True)
```

Comment:
`overlay_count` and `work_path` are only needed in the kernel case. So I suggest that we move these 3 lines into the kernel case below (inside `else` of `if use_fuse...`). But `temp_path` is always required.

Reply:
The latest commit moved the `temp_path` line away but kept the `overlay_count += 1`. I realize that GitHub shows this incorrectly, I tried to comment on all four lines, but GitHub attached the comment only to the last three lines. Sorry.